pkg/swagger/swagger.go

```diff
@@ -89,8 +89,8 @@ func Run(api, outputDir string) error {
 		},
 	}
 
-	// for installaOpenShiftVerions
-	if g.installableOpenShiftVersion {
+	// for installaVerions available for ARO
```

Comment:
nit: minor typo, IMO this comment is not needed
```suggestion
	// for installableVerions available for ARO
```

Reply:
done